specification/machinelearningservices/resource-manager/Microsoft.MachineLearningServices/preview/2020-05-01-preview/examples/createOrUpdateDeployment.json

```diff
@@ -0,0 +1,67 @@
+{
+  "parameters": {
+    "subscriptionId": "34adfa4f-cedf-4dc0-ba29-b6d1a69ab345",
+    "resourceGroupName": "testrg123",
+    "workspaceName": "workspaces123",
+    "endpointName": "testendpoint",
+    "deploymentName": "testdeployment",
+    "api-version": "2020-05-01-preview",
+    "parameters": {
+      "modelUris": [
+        "modelURL?"
+      ],
+      "skuSize": "Standard_DS2_V2",
+      "weight": 100,
+      "scaleSettings": {
+        "minimum": 1,
+        "maximum": 5
+      },
+      "environmentConfigUri": "EnvConfigURL?",
+      "codeAssetId": "CodeAssetURL?",
+      "vnet": {
+        "id": "subscriptions/34adfa4f-cedf-4dc0-ba29-b6d1a69ab345/resourceGroups/testrg123/providers/Microsoft.Network/virtualNetworks/testvnet",
+        "subnetName": "default"
+      }
+    }  
+  },
+  "responses": {
+    "200": {
+      "body": {
+        "id": "subscriptions/34adfa4f-cedf-4dc0-ba29-b6d1a69ab345/resourceGroups/testrg123/providers/Microsoft.MachineLearningServices/workspaces/workspaces123/endpoints/testendpoint/deployments/testdeployment",
+        "name": "testdeployment",
+        "identity": "",
+        "location": "westus2euap",
+        "type": "Microsoft.MachineLearningServices/workspaces/endpoints/deployments",
+        "sku": {
+          "name": "Basic_A0",
+          "tier": "Basic",
+          "capacity": 5
+        },
+        "properties": {
+          "modelIds": [
+            "modelURL?"
+          ],
+          "scaleSettings": {
+            "minimum": 1,
+            "maximum": 5
+          },
+          "weight": 100,
+          "provisioningState": "Succeeded",
+          "environmentConfigId": "EnvConfigURL?",
+          "codeAssetId": "CodeAssetURL?",
+          "vnet": {
+            "id": "subscriptions/34adfa4f-cedf-4dc0-ba29-b6d1a69ab345/resourceGroups/testrg123/providers/Microsoft.Network/virtualNetworks/testvnet",
+            "name": "testvnet",
```

Comment:
name and type required in request?

Reply:
This is the response. request asks for vnet resourceid and subnetName